src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/DeliverySessionServiceTest.kt

```diff
@@ -355,18 +395,6 @@ class DeliverySessionServiceTest @Autowired constructor(
       }
       assertThat(error.message).contains("No Delivery Session Appointment found")
     }
-
-    @Test
-    fun `expect failure when appointment is in the future`() {
-      val deliverySession = deliverySessionFactory.createScheduled(appointmentTime = OffsetDateTime.now().plusMonths(2))
-      val referral = deliverySession.referral
-      val appointment = deliverySession.currentAppointment!!
-
-      val error = assertThrows<ResponseStatusException> {
-        deliverySessionService.recordAppointmentAttendance(referral.id, appointment.id, defaultUser, Attended.YES, "additionalInformation")
-      }
-      assertThat(error.message).contains("Cannot submit feedback for a future appointment [id=${appointment.id}]")
```

Comment:
Is this assertion no longer relevant?

Reply:
As we need this feature, please bring this back

Reply:
reverted